### PR TITLE
Miklasr/18 document rpi setup

### LINKED ISSUE #18: Document RPi setup
Write initial documentation on how to set up the Raspberry Pi for this project, including a bill of materials. This can be updated in the future, as needed.

### PR DESCRIPTION
**Pull request by miklasr**
Initial documentation to setup the RPi and a bill of materials

Closes #18

### CODE REVIEW

File: docs/components.md [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
+- Optionally, a case for the Raspberry Pi and camera
+
+We estimate that this will cost around £150 for all components.
+
+## Assembly
+This section covers the hardware assembly. For software setup checkout the [software setup guide (TBC)]().
+
+1. Attach the camera module to the CSI port. This is the slot closest to the Ethernet (or furthest from the SD card). Lift up the plastic toggle and slide in the camera ribbon cable. Ensure the contacts on the cable face in the correct direction. Push down the toggle softly to fix the ribbon cable in place.
+1. Attach the screen or monitor. You will either attach this via a HDMI port (on the side) or the other ribbon cable port.
+1. Insert the flashed SD card (see [here(TBC)]() for insturctions on flashing).
```
Comment (Ashwin-MJ) [inline docs/components.md]: Minor things - these are all numbered step "1" and "insturctions" -> "instructions"

Comment (miklasr) [reply docs/components.md]: Typo fixed. I think in Markdown you're supposed to number by starting all lines with `1.` and then it automatically puts in the correct numbering. That stops any errors were lines are moved or removed causing the numbering to in the wrong order.

Comment (Ashwin-MJ) [reply docs/components.md]: Ooh okay didn't actually know that thanks!

**Review by Ashwin-MJ — APPROVED** [review]
Looks like a good starting point to me!

**Pull request merged**